pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.poetry]
+name = "opendis"
+version = "0.1.0"
+description = ""
+authors = ["Scott Robertson <scott.peter.robertson@gmail.com>"]
```

Comment:
This was a default, we can change this to the original authors of the package.

Reply:
Yes let's change to the default authors in setup

Reply:
updated